app/components/primer/flex_component.rb

```diff
@@ -32,6 +35,40 @@ class FlexComponent < Primer::Component
     DEFAULT_DIRECTION = nil
     ALLOWED_DIRECTIONS = [DEFAULT_DIRECTION, :column, :column_reverse, :row, :row_reverse].freeze
 
+    # @example 100|Default
+    #   <%= render(Primer::FlexComponent.new(bg: :gray)) do %>
+    #     <div class="p-5 border bg-gray-light">Item 1</div>
+    #     <div class="p-5 border bg-gray-light">Item 2</div>
+    #     <div class="p-5 border bg-gray-light">Item 3</div>
+    #   <% end %>
+    #
+    # @example 100|Justify center
+    #   <%= render(Primer::FlexComponent.new(justify_content: :center, bg: :gray)) do %>
+    #     <div class="p-5 border bg-gray-light">Item 1</div>
+    #     <div class="p-5 border bg-gray-light">Item 2</div>
+    #     <div class="p-5 border bg-gray-light">Item 3</div>
+    #   <% end %>
+    #
+    # @example 100|Align end
+    #   <%= render(Primer::FlexComponent.new(align_items: :end, bg: :gray)) do %>
+    #     <div class="p-5 border bg-gray-light">Item 1</div>
+    #     <div class="p-5 border bg-gray-light">Item 2</div>
+    #     <div class="p-5 border bg-gray-light">Item 3</div>
+    #   <% end %>
+    #
+    # @example 100|Direction column
+    #   <%= render(Primer::FlexComponent.new(direction: :column, bg: :gray)) do %>
+    #     <div class="p-5 border bg-gray-light">Item 1</div>
+    #     <div class="p-5 border bg-gray-light">Item 2</div>
+    #     <div class="p-5 border bg-gray-light">Item 3</div>
+    #   <% end %>
+    #
+    # @param justify_content [Symbol] Use this param to distribute space between and around flex items along the main axis of the container. <%= one_of(Primer::FlexComponent::JUSTIFY_CONTENT_OPTIONS) %>
+    # @param inline [Boolean] Defaults to false.
+    # @param flex_wrap [Boolean] Defaults to nil.
```

Comment:
This is a boolean but also defaults to nil... what's the right thing here?

Reply:
@srt32 that's a good question. What do we do in Primer React?

Reply:
Good question!

I also just noticed this on https://primer.style/components/Flex

> Previously, a Flex.Item component was used for flex item specific properties; Box now contains all those properties and should be used in its place.

Maybe we should deprecate FlexItem?

I am having a hard time grokking the PRC (I'm new to the codebase) but it looks to mainly delegate to `Box`, which delegates to styleSystem.

The API though is directly 

```
<Flex flexWrap="nowrap">
```

Reply:
@emplums what do you think? I'd generally lean towards deprecating all of the Flex-related components and just encourage the usage of Box with the flex-related system arguments.

Reply:
@joelhawksley the proposed game plan for PRC is that `Box` and `Flex` will have all the same properties, the only difference is `Flex` will have `display: flex` set by default. The idea is that it will be handy shortcut from having to type `display: flex` over and over, and will still keep the ease of grokking the code to see exactly where flex is being used